src/main.py

```diff
@@ -377,8 +378,11 @@ def main():
             if len(times) > 0:
                 logger(f"Average time: {sum(times) / len(times)}", 0)
 
-            logger(f"Looping in {sleep_duration}")
-            sleep(sleep_duration)
+            if should_loop:

```

Comment:
Instead of if should_loop else break i think it would be easier to understand to do something like this prior to the logger/sleep information.
```python
if run_once:
    break
```